src/utils.ts

```diff
@@ -25,6 +55,21 @@ export function parseUrl(url: string): Url {
   return urlObject;
 }
 
+/**
+ * Ensure path separators of the posix forward-slash form when
+ * running on environment (Windows) where path separator is a
+ * backslash.  Backslash *is* a valid filename character in a
+ * posix environment, so this function is a no-op there.
+ *
+ * @param pathname - file path to transform.
+ */
+export function posixify(pathname: string): string {
```

Comment:
The thing about windows is that it is presently agnostic about '/' and '\' and generally treats them the same, so we fully expect that they can come in already as '/'.

I don't perceive it as lossy for us to flip them around.